frontend/src/app/commonComponents/Required.tsx

```diff
@@ -1,7 +1,10 @@
 import React from "react";
 
-const Required: React.FC<{}> = () => (
-  <span style={{ color: "#E35024" }}>*</span>
+const Required = (props: { label?: React.ReactNode }) => (
+  <>
+    {props.label} <span className="usa-sr-only">Required</span>
```

Comment:
Another 508 thing. Screen readers will read "Required" on the labels of fields that are required.